---
layout: page
title: Citations
group: navigation
---

__Please cite our <a href="https://doi.org/10.1093/gigascience/gix119" target="_blank">paper</a> if you use the funRiceGenes dataset in your study!__  

**Publications that cite the funRiceGenes database:**  

1. Zong et al. [Feedback Regulation of ABA Signaling and Biosynthesis by a bZIP Transcription Factor Targets Drought-Resistance-Related Genes](http://www.plantphysiol.org/content/early/2016/06/20/pp.16.00469.abstract), 2016, Plant physiology.
2. Daware et al. [An Efficient Strategy Combining SSR Markers- and Advanced QTL-seq-driven QTL Mapping Unravels Candidate Genes Regulating Grain Weight in Rice](http://journal.frontiersin.org/article/10.3389/fpls.2016.01535/full), 2016, Frontiers in plant science.
3. Badoni et al. [Genome-wide generation and use of informative intron-spanning and intron-length polymorphism markers for high-throughput genetic analysis in rice](http://www.nature.com/articles/srep23765), 2016, Scientific reports.
4. Das et al. [Revisiting the decoded genomes to promptly reveal their genomic perspectives](http://www.currentscience.ac.in/Volumes/112/02/0279.pdf), 2017, Current Science.
5. Dingkuhn et al. [Crop-model assisted phenomics and genome-wide association study for climate adaptation of indica rice. 2. Thermal stress and spikelet sterility](https://doi.org/10.1093/jxb/erx250), 2017, Journal of Experimental Botany.
6. Golicz et al. [MCRiceRepGP: a framework for identification of sexual reproduction associated coding and lincRNA genes in rice](https://onlinelibrary.wiley.com/doi/abs/10.1111/tpj.14019), 2018, Plant Journal.
7. Gianinetti et al. [Seed Dormancy Involves a Transcriptional Program That Supports Early Plastid Functionality during Imbibition](http://www.mdpi.com/2223-7747/7/2/35/htm), 2018, Plants.
8. Liao et al. [A comprehensive review of web-based resources of non-coding RNAs for plant science research](http://www.ijbs.com/v14p0819), 2018, Int J Biol Sci.
9. Wing et al. [The rice genome revolution: from an ancient grain to Green Super Rice](https://www.nature.com/articles/s41576-018-0024-z), 2018, Nature Reviews Genetics.
10. Kim Nhung TA et al. [A genome-wide association study using a Vietnamese landrace panel of rice (Oryza sativa) reveals new QTLs controlling panicle morphological traits](https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-018-1504-1), 2018, BMC Plant Biology.
11. Hong et al. [Infrastructures of systems biology that facilitate functional genomic study in rice](https://link.springer.com/article/10.1186/s12284-019-0276-z), 2019, Rice.
12. Borrill et al. [Identification of transcription factors regulating senescence in wheat through gene regulatory network modelling](http://www.plantphysiol.org/content/early/2019/05/07/pp.19.00380.abstract), 2019, Plant physiology.
13. Yoo et al. [A Systematic View Exploring the Role of Chloroplasts in Plant Abiotic Stress Responses](https://www.hindawi.com/journals/bmri/2019/6534745/), 2019, BioMed Research International.
14. Ram et al. [Insertional Mutagenesis Approaches and Their Use in Rice for Functional Genomics](https://www.mdpi.com/2223-7747/8/9/310), 2019, Plants.
15. Yuan et al. [Rice Protocol eBook facilitates rice research](http://www.plant-physiology.com/upload/file/201812280935448659.pdf), 2018, Plant Physiology Journal.
16. Dhaka et al. [Small RNA profiling from meiotic and post-meiotic anthers reveals prospective miRNA-target modules for engineering male fertility in sorghum](https://www.sciencedirect.com/science/article/pii/S0888754319302927), 2019, Genomics.
17. Chandran et al. [A web-based tool for the prediction of rice transcription factor function](https://academic.oup.com/database/article/doi/10.1093/database/baz061/5511305), 2019, Database.
18. Peng et al. [MBKbase for rice: an integrated omics knowledgebase for molecular breeding in rice](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz921/5590660), 2019, Nucleic Acids Research.
19. Liu et al. [Identification of genes for salt tolerance and yield-related traits in rice plants grown hydroponically and under saline field conditions by genome-wide association study](https://link.springer.com/article/10.1186/s12284-019-0349-z), 2019, Rice.
20. Dhaka et al. [Transcriptional trajectories of anther development provide candidates for engineering male fertility in sorghum](https://www.nature.com/articles/s41598-020-57717-0), 2020, Scientific Reports.
21. Yoo et al. [Fast Track to Discover Novel Promoters in Rice](https://www.mdpi.com/2223-7747/9/1/125), 2020, Plants.
22. Yang et al. [Crop Phenomics and High-Throughput Phenotyping: Past Decades, Current Challenges, and Future Perspectives](https://www.sciencedirect.com/science/article/pii/S1674205220300083), 2020, Molecular Plant.
23. DO et al. [Candidate gene prioritization using graph embedding](https://www.biorxiv.org/content/10.1101/2020.02.03.927913v1), 2020, BioRxiv.
24. Jia et al. [Pleiotropic changes revealed by in situ recovery of the semi-dwarf gene sd1 in rice](https://doi.org/10.1016/j.jplph.2020.153141), 2020, Journal of Plant Physiology.
25. Lu et al. [A direct PCR–based SNP marker–assisted selection system (D-MAS) for different crops](https://link.springer.com/article/10.1007/s11032-019-1091-3), 2020, Molecular Breeding.
26. Cubry et al. [Genome wide association study pinpoints key agronomic QTLs in African rice Oryza glaberrima](https://www.biorxiv.org/content/10.1101/2020.01.07.897298v1.full), 2020, BioRxiv.
27. Mohammed Abdullah Abdulraheem Ghaleb et al. [Heterosis analysis and underlying molecular regulatory mechanism in a wide-compatible neo-tetraploid rice line with long panicles](https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-020-2291-z), 2020, BMC Plant Biology.
